changelog/august2023/2023-08-22-cockpit-changed-cockpit-feature-request-is.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Cockpit feature request is completed
+status: changed
+author:
+    fullname: 'Join the #[xxx] channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-08-22
+category: observability
+product: cockpit
+---
+
+We are happy to announce that we closed one of the most wanted feature : https://feature-request.scaleway.com/posts/38/log-centralization-and-management at Scaleway (open since Jan. 21)
```

Comment:
```suggestion
We are happy to announce that we closed one of the [most wanted features]https://feature-request.scaleway.com/posts/38/log-centralization-and-management  at Scaleway (open since Jan. 21).
```